Debug App/Sources/View Controllers/MerchantSessionAndSettingsViewController.swift

```diff
@@ -224,6 +224,7 @@ class MerchantSessionAndSettingsViewController: UIViewController {
             environmentSegmentedControl.selectedSegmentIndex = 1
         }
         
+        customDefinedApiKey = "ccee2be8-df08-4c79-856d-40085349e9da"
         self.apiKeyTextField.text = customDefinedApiKey
```

Comment:
These three changes:
- Changing default env to .staging
- Changing default session currency to .AED
- Hardcoding an apiKey

Are they intended?

Reply:
oooops!